python/tk_desktop/site_communication.py

```diff
@@ -42,11 +43,15 @@ def _create_proxy(self, pipe, authkey):
         CommunicationBase._create_proxy(self, pipe, authkey)
         self.proxy_created.emit()
 
+    @property
+    def server_pipes(self):
+        return self._msg_server.pipes
+
     def start_server(self):
         """
         Sets up a server to communicate with the background process.
         """
-        self._create_server()
+        self._create_server(DualRPCServer)
```

Comment:
Ah is this why the `_create_server` is setup to be passed a server factory object, since it needs to be able to be passed this class?

Reply:
Yup!